ament_cmake_nose/cmake/ament_add_nose_test.cmake

```diff
@@ -133,5 +133,3 @@ function(_ament_add_nose_test testname path)
     LABELS "nose"
   )
 endfunction()
-
-# TODO provide function to register all found tests separately
```

Comment:
I'm not sure what this TODO means exactly, and where has the function it refers to been implemented. Can you provide a pointer to that function ?